meta/FlexibleSUSY.m

```diff
@@ -2057,8 +2057,13 @@ corresponding tadpole is real or imaginary (only in models with CP
                        https://mathematica.stackexchange.com/questions/11595/package-found-with-needs-but-not-with-parallelneeds *)
                     $Path = contentOfPath;
                     (* don't pollute terminal with SARAH initialization message *)
-                    Block[{Print},
+                    Block[{Print,workingDirectory = Directory[]},
                        << SARAH`;
+                       SARAH`SARAH[OutputDirectory] = FileNameJoin[{workingDirectory, "Output"}];
+                       SARAH`SARAH[InputDirectories] = {
+                          FileNameJoin[{workingDirectory, "sarah"}],
+                          ToFileName[{$sarahDir, "Models"}]
+                       };
                        Start@modelName;
```

Comment:
One question regarding `Start@modelName;`

SARAH also has so-called "sub-models". An example is the `MSSM/CKM` or `MSSM/CPV`. To my knowledge these models are loaded as:
~~~.m
Start["MSSM", "CPV"];
~~~
I'm wondering if the above call of `Start` works also with sub-models. Did you test this at some point?

Reply:
In the `MSSMCPV` `modelName` is `MSSM/CPV`.  The old syntax `Start[MSSM/CPV]` still works so there is no problem there, at least for now.

Reply:
Ah, ok, thanks for checking!